platform/viewer/src/components/Snackbar/Snackbar.css

```diff
@@ -0,0 +1,235 @@
+.sb-container {
+  position: fixed;
+  width: 420px;
+  padding: 20px;
+  z-index: 9998;
```

Comment:
If possible, it might be nice to also set/pull this as a css-variable, so we have better visibility into how our application is "layered"

Reply:
Cool! I've added in the `:root` as a CSS variable. I did a quick search in the application for `z-index` and there are a lot of different values. Maybe it would be good to sometime take them all and put in the CSS variable as well 😊